infra/src/main/kotlin/fi/fta/geoviite/infra/ratko/RatkoService.kt

```diff
@@ -102,6 +110,74 @@ class RatkoService @Autowired constructor(
         }
     }
 
+    fun manageRatkoBulkTransfers() {
+        lockDao.runWithLock(DatabaseLock.RATKO_BULK_TRANSFER, databaseLockDuration) {
```

Comment:
En ole ihan varma mutta tuntuisi että tämä olisi ehkä turvallisempaa pitää saman lukon takan muun ratko pushin kanssa ettei niitä mene päälekäin. Nuo molemmat siirrot voisi jopa laittaa ketjuun saman lukko-blokin sisään. Vai oliko tätä tarkoitus kutsua jostain erikseen?